undertow-servlet/src/main/java/org/wildfly/elytron/web/undertow/server/servlet/ElytronHttpServletExchange.java

```diff
@@ -337,11 +337,6 @@ private FormResponseWrapper(final HttpServletResponse wrapped) {
             super(wrapped);
         }
 
-        @Override
```

Comment:
Just noting that as mentioned on chat, one option could be to re-add this method but remove the `@Override` annotation.

Reply:
I updated it accordingly @fjuma .

Reply:
@ropalka We discussed this more today and decided to create a 4.x branch for Elytron Web. I've updated this PR so the base branch is now set to 4.x.

This means that we can now remove the `setStatus` method as you had done before.

Reply:
setStatus() method was removed